web_src/js/features/repo-issue.js

```diff
@@ -455,6 +455,12 @@ export function initRepoPullRequestReview() {
     const _promise = initComboMarkdownEditor($reviewBox.find('.combo-markdown-editor'));
   }
 
+  // set the value of type to the button's type value for the submit review panel
+  for (const btn of $reviewBox.find('.btn-submit')) {
+    $(btn).on('click', function () {
+      $reviewBox.find('input[name="type"]').attr('value', $(this).attr('value'));
```

Comment:
I do not think it needs to be so.

You can add the submitter's value to the FormData